src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -531,45 +531,23 @@ public Collection<NodeProvisioner.PlannedNode> provision(@NonNull final Cloud.Cl
             Set<String> allInProvisioning = InProvisioning.getAllInProvisioning(label); // Nodes being launched
             LOGGER.log(Level.FINE, () -> "In provisioning : " + allInProvisioning);
             int toBeProvisioned = Math.max(0, excessWorkload - allInProvisioning.size());
-            Set<Node> nodes = getNodes(label);
-            int currentExecutorsCount = (int) nodes.stream().filter(KubernetesSlave.class::isInstance).map(Node::getNumExecutors).count();
-            int launchingExecutorsCount = (int) nodes.stream().filter(KubernetesSlave.class::isInstance).filter(n -> n.toComputer() != null && ((KubernetesComputer)n.toComputer()).isLaunching()).map(Node::getNumExecutors).count();
-            LOGGER.log(Level.INFO, "Label \"{0}\" excess workload: {1}," +
-                    " executors: {2}," +
-                    " plannedCapacity: {3}," +
-                            " launching: {4}",
-                    new Object[] {label, toBeProvisioned, currentExecutorsCount, plannedCapacity, launchingExecutorsCount});
             List<NodeProvisioner.PlannedNode> plannedNodes = new ArrayList<>();
-            List<Pod> pods = getPodsWithLabels(namespace, getPodLabelsMap());
-            if (pods != null) {
-                // Count planned and executors not yet launching (e.g. everything that is not in kubernetes yet)
-                int plannedCount = plannedCapacity + currentExecutorsCount - launchingExecutorsCount;
```

Comment:
Unless I am missing something, with the deletion of this line just three months after https://github.com/jenkinsci/kubernetes-plugin/pull/835, it means that the API introduced in https://github.com/jenkinsci/jenkins/pull/4922 is no longer being used except for log messages.